api/proto/wso2/discovery/ga/api.proto

```diff
@@ -14,4 +14,5 @@ message Api {
   string apiUUID = 1;
   string revisionUUID = 2;
   string organizationUUID = 3;
+  string deployedEnv = 4;
```

Comment:
should deployedEnv be optional? (If it is not breaking the current implementation, it is okay to keep this field as it is IMO)

Reply:
I have locally tested this and seems it is optional by default. I tested this on a GA running with the older version (without having this field added) and a LA with the new version (with having this field added) and it was working without any issue. An empty value was received for the newly added field.

Also from the discussions [1] & [2] I think this behavior is correct for proto3.

[1] : https://stackoverflow.com/questions/31801257/why-required-and-optional-is-removed-in-protocol-buffers-3
[2] : https://github.com/protocolbuffers/protobuf/issues/2497#issuecomment-267422550